crates/database/src/orders.rs

```diff
@@ -1050,36 +1045,85 @@ mod tests {
         assert!(get_order(&mut db, 2).await.is_some());
     }
 
+    type Data = ([u8; 56], Address, DateTime<Utc>);
+    async fn user_orders(
+        ex: &mut PgConnection,
+        owner: &Address,
+        offset: i64,
+        limit: Option<i64>,
+    ) -> Vec<Data> {
+        super::user_orders(ex, owner, offset, limit)
+            .map(|o| {
+                let o = o.unwrap();
+                (o.uid.0, o.owner, o.creation_timestamp)
+            })
+            .collect::<Vec<_>>()
+            .await
+    }
+
     #[tokio::test]
     #[ignore]
-    async fn postgres_user_orders_performance() {
-        //The following test can be used as performanc etest, if the values for
-        // i and j are increased ->i=240 and j=1000 are reasonable values
+    async fn postgres_user_orders_performance_many_users_with_some_orders() {
+        // The following test can be used as performance test,
+        // if the values for i and j are increased ->i=100
+        // and j=1000 the query should still 10 ms
```

Comment:
I guess for test like these it makes sense to have criterion benchmarks. Then we could immediately get the min, max, avg of multiple test runs. I guess we could even include that in the CI system to fail a build if a PR causes a big performance regression. This doesn't have to happen in this PR but I think it would be valuable to experiment with that option at some point in time.

Reply:
This is a very good idea, maybe we don't need to fail the CI, maybe just post a comment whenever a regression happens.

Reply:
Yes, I think so too that we should do it at some point. But we should do it in another Pr.

>I guess we could even include that in the CI system to fail a build if a PR causes a big performance regression.

I think we can not do it every CI run, as the execution of loading all the orders into the database takes longer than 2 minutes.